spec/spec_helper.rb

```diff
@@ -5,6 +5,9 @@
 
 SimpleCov.start 'rails' do
   minimum_coverage 100
+  # Instagram is being decommissioned per
+  # https://cm-jira.usa.gov/browse/SRCH-50
+  add_filter %r{instagram}
 end
 
```

Comment:
Can you also add this comment in the rspec and code. This way another developer would know this is dead code till we cleanup.